tests/test_bonded.py

```diff
@@ -167,6 +168,58 @@ def test_flat_bottom_bond(self, n_particles=64, n_bonds=35, dim=3):
             np.testing.assert_array_equal(test_du_dx, test_du_dx_rev)
             np.testing.assert_array_equal(test_du_dp, test_du_dp_rev)
 
+    def test_log_flat_bottom_bond(self, n_particles=64, n_bonds=35, dim=3):
+        """Randomly connect pairs of particles, then validate the resulting LogFlatBottomBond force"""
+        np.random.seed(2022)
+
+        # TODO(deboggle) : reduce code duplication between HarmonicBond and LogFlatBottomBond
```

Comment:
```suggestion
        # TODO(deboggle) : reduce code duplication between HarmonicBond, FlatBottomBond, and LogFlatBottomBond
```

(implementation more closely follows https://github.com/proteneer/timemachine/pull/764 than HarmonicBond -- both introduce a little duplication / variation, could possibly refactor `GenericBondForce` or similar to template on u(r, p), du_dr(r, p), du_dp(r, p) or something in the future)